tools/emscripten.py

```diff
@@ -778,18 +778,15 @@ def create_em_js(metadata):
 def add_standard_wasm_imports(send_items_map):
   extra_sent_items = []
 
-  if settings.IMPORTED_MEMORY:
-    memory_import = 'wasmMemory'
-    if settings.MODULARIZE and settings.PTHREADS:
-      # Pthreads assign wasmMemory in their worker startup. In MODULARIZE mode, they cannot assign inside the
-      # Module scope, so lookup via Module as well.
-      memory_import += " || Module['wasmMemory']"
-    send_items_map['memory'] = memory_import
-
   if settings.SAFE_HEAP:
     extra_sent_items.append('segfault')
     extra_sent_items.append('alignfault')
 
+  # Special case for importing memory and table
+  # TODO(sbc): can we make these into normal library symbols?
```

Comment:
Are there symbol types for memories?

Reply:
Yes, we could add memories and tables to this list: https://github.com/emscripten-core/emscripten/blob/b76078830dca3d11ce08d43b3e6a54dc8f193af6/tools/link.py#L2904

My initial attempt to do that didn't work but I hope to figure it out at some point.